managed/src/main/java/com/yugabyte/yw/commissioner/tasks/DestroyUniverse.java

```diff
@@ -55,6 +66,53 @@ public void run() {
         universe = lockUniverseForUpdate(-1 , true);
       }
 
+      if (params().isDeleteBackups) {
+        List<Backup> backupList = Backup.fetchByUniverseUUID(params().customerUUID, universe.universeUUID);
+        for (Backup backup : backupList) {
+          try {
+            backup = Backup.get(params().customerUUID, backup.backupUUID);
+            if (backup.state != Backup.BackupState.Completed) {
+              LOG.error("Cannot delete backup in any other state other than completed.");
```

Comment:
Should we insert `continue` here / after the log?

Reply:
yeah, we should.

Reply:
Should we raise an exception once we get a backup that is still in progress while deleting the universe? ( guess universe lock should take care of it but still )

Reply:
Nope, in the case of a force delete, this can be a legitimate issue, coz in that case we don't take care of the locks. Also, for backups, the universe isn't actually locked while the actual bulk of the work is happening. Maybe we can add a check to see if there are backups in progress, and if so, we can just a) either wait it out, b) just fail that subtask and try deleting it anyway.

Reply:
@Arnav15 waiting for the backup to complete or fail ( other than in progress ) before moving to the next code blocks makes sense to me. LMK if I can go ahead with the change.

Reply:
![Screenshot from 2021-04-15 21-54-55](https://user-images.githubusercontent.com/50577199/114904433-7cd94080-9e35-11eb-8577-20b561c6bb2f.png)
If we make it a subtask , this page looks like this.. Is that fine? @SergeyPotachev @Arnav15

Reply:
@sb-yb

Reply:
I think its because it does not have subtasks and percentage is calculated as num_completed/num_total subtasks. 
May be if num_total  is 0 we should set percentage to 100 instead of 0. WDYT @Arnav15 @SergeyPotachev ?

Reply:
@sb-yb Yes, makes sense.

Reply:
`public double getPercentCompleted() {
    int numSubtasks = TaskInfo.find.query().where()
      .eq("parent_uuid", getTaskUUID())
      .findCount();
    if (numSubtasks == 0) {
      return 0.0;
    }
    int numSubtasksCompleted = TaskInfo.find.query().where()
      .eq("parent_uuid", getTaskUUID())
      .eq("task_state", TaskInfo.State.Success)
      .findCount();
    return numSubtasksCompleted * 100.0 / numSubtasks;
  }`
  yes , but do we have to return 100 if no of the subtask is 0? ( does not make sense if it gets failed.. but even 0 not making sense when the task is completed successfully.

Reply:
yes , but do we have to return 100 if no of the subtask is 0? ( does not make sense if it gets failed.. but even 0 not making sense when the task is completed successfully.